.travis.yml

```diff
@@ -1,4 +1,13 @@
 language: scala
 scala:
-   - 2.11.11
-   - 2.12.6
\ No newline at end of file
+- 2.11.11
+- 2.12.6
+env:
+  global:
+#  Encrypted SONATYPE_USER, SONATYPE_PASSWORD, PGP_PASS
+  - secure: bJoASB6VNHk0DMK/AwvQj1Q72/P4jzt9Xu6uWHJrdSJIdkkgxxJ6BlXDHNqcnCn22rpW0/dr34pCtz4oNzZ8HNsGpVra5jc1YuACV3RLNEOmBELFuQI+Dik7jJLsjraQ9BNhAJ6qm3mu4CKV7yGRWRW8jmJ6SbQU4hAlO73M+RgeFJRf35Ly2ZtlSfm69q9bbcudN35sgyrUyYqIc3S7chEWgSrnfluoO+1r5LjjNanwJfIvfyFNNdTIczujJELKw7tNceO9ouaLcxsd6+zJur06U53RrYPvmOLbpv4/F6eDzTX/popeV5grhil2DiuKwLa5kjVH1JwMhbXdLmDUk2SHYf2gbS9E11F4whr29EkvI9Ql0hmqNijqyMO13qr7U1kfrF8aAqhMPquPquPLC3wD74SNj30cpAgO62r4HlbDmYnauJ7srqtsmrUXFllvdp1pJltljWqQcHfI/QWRzYJS3RqcIFUCyIdTMm+hcHYmh9GFsJDn5Hfrf2vg5fQFwA0Ru88Hyg26dlJq/YHSLucG8HPPfzSJ8laVVuN3zRQNNbOkvRqTx4dSbzeDF1ndoSTMXQqzH8BoOGMjWQgVJWmdrshWJGT3o0OSwv3LPN9HSJ6kfbp2Nd4nRRuM9SDMKdxjUDJV/EOTn390caEoimJ794VoppZZZwF3OC+/r34=
```

Comment:
I’d double check this works, because in my experience I had to do something like:

```
FOO=foo BAR=bar BAZ=baz
```

and encrypt a single string to get all three environment variables.

Reply:
ah ok thanks